sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1097,3 +1097,28 @@ def test_vectorizers_invalid_ngram_range(vec):
     if isinstance(vec, HashingVectorizer):
         assert_raise_message(
             ValueError, message, vec.transform, ["good news everyone"])
+
+
+@pytest.mark.parametrize("vec", [CountVectorizer()])
```

Comment:
I think there is no need to parametrize it, since we are only checking the `CountVectorizer` (which should be sufficient).

Reply:
Yeah, I wasn't sure at first if we needed to test this with other vectorizers, but I don't think so.